apps/usb_probe.py

```diff
@@ -168,15 +168,34 @@ def is_bluetooth_hci(device):
 # -----------------------------------------------------------------------------
 @click.command()
 @click.option('--verbose', is_flag=True, default=False, help='Print more details')
-def main(verbose):
+@click.option(
+    '--interactive',
+    is_flag=True,
+    default=False,
+    help='Detech a device by unplugging / replugging it in',
+)
+def main(verbose, interactive):
     logging.basicConfig(level=os.environ.get('BUMBLE_LOGLEVEL', 'WARNING').upper())
 
     load_libusb()
     with usb1.USBContext() as context:
         bluetooth_device_count = 0
         devices = {}
 
-        for device in context.getDeviceIterator(skip_on_error=True):
+        if interactive:
+            input("Unplug your device and press the enter key!")
```

Comment:
It would be nice to also support USB hotplug events to re-scan automatically when a device is removed/added, so you don't have to type anything on the keyboard.